evadb/binder/binder_utils.py

```diff
@@ -339,3 +340,18 @@ def get_column_definition_from_select_target_list(
                 )
             )
     return binded_col_list
+
+
+def drop_row_id_from_target_list(
+    target_list: List[AbstractExpression],
+) -> List[AbstractExpression]:
+    """
+    This function is intended to be used by CREATE UDF FROM (SELECT * FROM ...) and CREATE TABLE AS SELECT * FROM ... to exclude the row_id column.
```

Comment:
Is `CREATE TABLE AS SELECT * FROM ...` also covered in this PR?

Reply:
No, it is not covered in this PR. This PR only contains the support for Model training.